CHANGELOG.md

```diff
@@ -19,6 +19,12 @@ Please add to the relevant subsections under Unreleased below on every PR where
 
 - Detect the notebook name in VS Code's built-in jupyter server (@dmitryduev in https://github.com/wandb/wandb/pull/8311)
 
+### Changed
+
+- The `wandb` package now uses the `wandb-core` binary as the default runtime service. To revert to the legacy behavior,
```

Comment:
"`wandb-core` binary" feels like an implementation detail. We've referred to this as "core" 
and "wandb-core" before in the changelog and communications, so how about

> The new "core" backend, previously activated using `wandb.require("core")`, is now used by default. [the rest]